x-pack/plugin/src/test/resources/rest-api-spec/test/rollup/get_rollup_index_caps.yml

```diff
@@ -288,6 +288,105 @@ setup:
                   - agg: "max"
                   - agg: "sum"
 
+---
+"Verify job caps by rollup index comma delimited list":
+
+- do:
```

Comment:
You'll need to add a skip section to not run this test on version that don't have the fix:
````
  - skip:
      version: " - 6.99.99"
      reason: "comma delimited index support was fixed in 7.0"
````
After the backport to 6x we'll change the skip version to 6.4.99.